### PR TITLE
Update production-offline-deployment-using-tiup.md

### PR DESCRIPTION
**Pull request by lucklove**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
LGTM

**Review by july2993 — COMMENTED** [review]
suggest add in "2. 部署离线环境 TiUP 组件" part

**Comment by lucklove** [comment]
> suggest add in "2. 部署离线环境 TiUP 组件" part

Done

File: production-offline-deployment-using-tiup.md [hunk 1]
```diff
@@ -70,11 +70,13 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/']
 {{< copyable "shell-regular" >}}
 
 ```bash
-tar xzvf tidb-community-server-${version}-linux-amd64.tar.gz &&
+tar xzvf tidb-community-server-${version}-linux-amd64.tar.gz
 sh tidb-community-server-${version}-linux-amd64/local_install.sh
 source /home/tidb/.bash_profile
 ```
 
+在 local_install.sh 中，会自动执行 `tiup mirror set tidb-community-server-${version}-linux-amd64` 将当前镜像地址设置为 `tidb-community-server-${version}-linux-amd64`。若有多个离线包，后续可以通过手动执行 `tiup mirror set  <mirror-dir>` 的方式来切换。
```
Comment (july2993) [inline production-offline-deployment-using-tiup.md:78]: 所以并不需要自己 mirror set? 后续如果又切到其它 mirror 才需要？

Comment (lucklove) [reply production-offline-deployment-using-tiup.md:78]: 是不需要自己执行 mirror set 的，mirror set 可以解决错误的 root.json 的问题，但是没法解决 root.json 缺失的问题，因为我们有个地方需要在全局读 root.json

Comment (lucklove) [reply production-offline-deployment-using-tiup.md:78]: 实际上 root.json 缺失了，我们缺失可以认为当前的状态是“未安装”

Comment (lonng) [inline production-offline-deployment-using-tiup.md:78]:
```suggestion
在 local_install.sh 中，会自动执行 `tiup mirror set tidb-community-server-${version}-linux-amd64` 将当前镜像地址设置为 `tidb-community-server-${version}-linux-amd64`。

> 若需将镜像切换到其他目录，可以通过手动执行 `tiup mirror set  <mirror-dir>` 方式切换。
```

**Review by lonng — COMMENTED** [review]
LGTM

File: production-offline-deployment-using-tiup.md [hunk 2]
```diff
@@ -70,11 +70,15 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/']
 {{< copyable "shell-regular" >}}
 
 ```bash
-tar xzvf tidb-community-server-${version}-linux-amd64.tar.gz &&
+tar xzvf tidb-community-server-${version}-linux-amd64.tar.gz
 sh tidb-community-server-${version}-linux-amd64/local_install.sh
 source /home/tidb/.bash_profile
 ```
 
+在 local_install.sh 中，会自动执行 `tiup mirror set tidb-community-server-${version}-linux-amd64` 将当前镜像地址设置为 `tidb-community-server-${version}-linux-amd64`。
```
Comment (TomShawn) [inline production-offline-deployment-using-tiup.md:78]:
```suggestion
`local_install.sh` 会自动执行 `tiup mirror set tidb-community-server-${version}-linux-amd64` 命令将当前镜像地址设置为 `tidb-community-server-${version}-linux-amd64`。
```

File: production-offline-deployment-using-tiup.md [hunk 3]
```diff
@@ -70,11 +70,15 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/']
...
+
+> 若需将镜像切换到其他目录，可以通过手动执行 `tiup mirror set <mirror-dir>` 方式切换。
```
Comment (TomShawn) [inline production-offline-deployment-using-tiup.md:80]:
```suggestion
若需将镜像切换到其他目录，可以通过手动执行 `tiup mirror set <mirror-dir>` 进行切换。
```

File: production-offline-deployment-using-tiup.md [hunk 4]
```diff
@@ -227,13 +231,10 @@ alertmanager_servers:
 
 ## 5. 部署 TiDB 集群
```
Comment (TomShawn) [inline production-offline-deployment-using-tiup.md:233]:
```suggestion

执行以下命令来部署 TiDB 集群。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**